packages/nextjs/scaffold.config.ts

```diff
@@ -9,9 +10,26 @@ export type ScaffoldConfig = {
   walletAutoConnect: boolean;
 };
 
+const nillionTestnet: chains.Chain = defineChain({
```

Comment:
I was looking for this and was stuck in wagmi land

Reply:
it's part of viem here: https://viem.sh/docs/chains/serializers#usage